recipes/intake-salesforce/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "intake-salesforce" %}
+{% set version = "0.0.9" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/intake-salesforce-{{ version }}.tar.gz
+  sha256: 5f0f8363710690c8310ef06c42ce1c7fefe9d8151e3e79c9b99b9bd7fc20e178
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - intake
+    - pandas
+    - python >=3.7
+    - simple-salesforce
+
+test:
+  imports:
+    - intake_salesforce
+    - tests
```

Comment:
This line passing means that it installs a `tests` package in the global namespace, which will collide with anything else that does that. Can you change upstream to replace `find_packages()` with just `['intake_salesforce']`?

Reply:
Thanks so much for checking! I just made the changes. Hope everything looks fine now. Please let me know I need to change anything else. Really appreciate your help :)